add floor divide

b-sai opened the pull request:
Close #9512

b-sai commented:
Close #9512

jkeane508 reviewed (changes requested):
Hey,

Thanks for submitting this PR

The test for this PR didn't run properly, but in the meantime, there are lint errors in the files relevant to this PR, if you haven't already you can install a pre-commit hook to prevent these in the future [here](https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit)

Will get back when the test runs successfully!

Happy Coding :)

b-sai commented:
Hey James,

Thanks for reviewing my PR. I believe the failing tests are not from my test cases. Running the tests locally (image below), I see that all 4 of them pass. I also fixed the lint errors. The current failed lint errors don't seem to be from any code that I have changed. Let me know if there's anything else I need to change.


![image](https://user-images.githubusercontent.com/67334664/211712253-ddaecd6e-8f02-4b4d-9028-706c22adc276.png)
